plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -186,6 +186,10 @@ func ClusterRoles() []rbac.ClusterRole {
 					"/openapi", "/openapi/*",
 					"/api", "/api/*",
 					"/apis", "/apis/*",
+					"/bulk", "/bulk/*",
```

Comment:
why is this a new top-level context root? I expected these operations to live within an API group

Reply:
Bulk is quite "special" api group. It doesn't contain any entities.
It doesn't support any existing verbs (get/list/watch etc).

> We will start with introducing a new dedicated API group:
> /apis/bulk.k8s.io/

I don't think it is good idea to serve wssocket on 'apis/bulk.k8s.io'.
Maybe we can use  '/apis/bulk.k8s.io/connect' (or something similar) where 'connect' is a fake resource (list/get/watch/etc are not supported). But I'm not sure that it will not break any existing clients/logic.  Also It should be handled differently in aggregator (as it just proxies all requests to another apiserver, which is not enough for bulk). So some 'if path == "bulk.k8s.io" need to be   added... Looks a bit hacky and unmaintainable.

Anyway I don't like to introduce any new top-level path but don't know a better place for it.
So I open for any suggestions :)